### PR TITLE
Sub

### PR DESCRIPTION
**Pull request by k12re**

### CODE REVIEW

**Comment by nerdychap** [comment]
- On your home page, make sure that your wall posts have spaces in between them, it looks cramped
- Profile page on smaller devices, your posts touch the edge of the device, consider adding some margin
- your create post input on the home page is bigger than the one on your home page, make sure everything is consistent
- Align the add file to the submit button
image

File: home.html [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+                  <label for="exampleFormControlTextarea1" class="form-label">Write something..</label>
+                  <textarea class="form-control" id="exampleFormControlTextarea1" rows="3"></textarea>
+                </div>
+                <button type="submit" class="btn btn-primary">Submit</button>
+                <i class="bi bi-paperclip"></i><span>Add file</span>
+              </form>
+            </div>
+          </section>
+          <section class="container">
+            <div class="row">
```
Comment (Fermain) [inline home.html]: The direct child of a `row` should always be a `col` type class.

File: home.html [hunk 2]
```diff
@@ -0,0 +1,130 @@
...
+              <h2>Wall posts</h2>
+                <div class="card p-4 mb-3 col-12 col-md-6 col-lg-4">
```
Comment (Fermain) [inline home.html]: Please use a code formatter to indent your code automatically.

File: index.html [hunk 3]
```diff
@@ -4,9 +4,39 @@
...
+    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap-icons@1.10.3/font/bootstrap-icons.css">
+    <title>Login</title>
 </head>
 <body>
-    <p>Hello!</p>
+  <img src="./img/megaphone-oleg-laptev-QRKJwE6yfJo-unsplash.jpg" class="position-fixed top-50 start-50 translate-middle" alt="An orange megaphone">
+    <main class="justify-content-center align-items-center mw-100"> 
+        <section class="container">
+          <div class="row align-items-center p-0 position-absolute top-50 start-50 translate-middle w-75">
+            <h1 class="text-center text-md-start h1 mb-5 fw-bold text-capitalized col-sm-12 col-md-8 position-relative">Megaphone Social</h1>
```
Comment (Fermain) [inline index.html]: Same here, child content of a row should always be a col type class

File: profile.html [hunk 4]
```diff
@@ -0,0 +1,187 @@
...
+                <button type="submit" class="btn btn-primary mt-3">Submit</button>
+                <i class="bi bi-paperclip"></i><span>Add file</span>
+              </form>
+            </div>
+          </div>
+        </section>
+        </section>
+        <section class="container mb-4">
+          <div class="row">
+                <h2>My posts</h2>
```
Comment (Fermain) [inline profile.html]: Same here, row/col

File: src/scss/_variables.scss [hunk 5]
```diff
@@ -0,0 +1,3 @@
+$primary: #ff7022;
```
Comment (Fermain) [inline src/scss/_variables.scss:1]: Although this meets requirements, this is close to the minimum amount of customisation possible. Your design doesn't require more than one special colour? You are happy with all other design choices made in Bootstrap?

**Review by Fermain — CHANGES_REQUESTED** [review]
This is good work that meets most of the requirements well. There are some issues with row/col to be corrected in order to pass.

Beyond that, I would suggest spending some more time customising SCSS variables. The power of Bootstrap is significant when used in this way and I do not see this being used to good effect in this project yet.

There are hundreds of lines of variables covering all of bootstrap's sizing, border, color, spacing variables. I strongly recommend getting to know some more of these.

File: home.html (hunk 2 above)
Comment (k12re) [reply home.html]: I've had this installed for a while, but I just found my formatter settings where set to null.. Thanks for making me investigate this further. Actually, when searching for format in settings, the default formatter doesn't show.. removed the last letter in search term, and there it is...

File: home.html (hunk 1 above)
Comment (k12re) [reply home.html]: Alright. I'll fix this. Thanks

File: index.html (hunk 3 above)
Comment (k12re) [reply index.html]: Fixed

File: profile.html (hunk 4 above)
Comment (k12re) [reply profile.html]: Done

File: src/scss/_variables.scss (hunk 5 above)
Comment (k12re) [reply src/scss/_variables.scss:1]: Yes, I will look into this. See if i can add some more tweaks throughout the weekend..

Thanks for good feedback! Have a great weekend..

**Pull request merged**